src/simple-todos/step05/imports/ui/App.jsx

```diff
@@ -4,21 +4,24 @@ import { TasksCollection } from '/imports/api/TasksCollection';
 import { Task } from './Task';
 import { TaskForm } from './TaskForm';
 
-const toggleChecked = ({ _id, isChecked }) => {
-  TasksCollection.update(_id, {
-    $set: {
-      isChecked: !isChecked,
-    },
-  });
-};
+const toggleChecked =
+  async ({ _id, isChecked }) => {
+    await TasksCollection.updateAsync(_id, {
+      $set: {
+        isChecked: !isChecked,
+      },
+    });
+  };
 
-const deleteTask = ({ _id }) => TasksCollection.remove(_id);
+const deleteTask =
+  async ({ _id }) => await TasksCollection.removeAsync(_id);
 
 export const App = () => {
-  const tasks = useTracker(() =>
-    TasksCollection.find({}, { sort: { createdAt: -1 } }).fetch()
+  const tasks = useTracker(async () =>
+    await TasksCollection.find({}, { sort: { createdAt: -1 } }).fetchAsync()
```

Comment:
I needed to stop this pr because useTracker could not handle async states at the time.

Now it can. We were planing on coming back on this one following the same style as the vue3 guide that was launched